rules/no-console-spaces.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+const getDocsUrl = require('./utils/get-docs-url');
+
+const getConsoleMethod = (node) => {
+	const methods = [
+		'log',
+		'warn',
+		'error',
```

Comment:
There are many more log methods: https://developers.google.com/web/tools/chrome-devtools/console/console-reference We only need to support the ones that accept multiple arguments.